### ISSUE TITLE
Last Node and Edge when export json

### ISSUE THREAD

**Issue by jiisoo** [ghaffarian/progex#9]
I export a json not a dot, and it generates a json file successfully.
However, when I try to read the json file using below codes, it shows an error `JSONDecodeError: Expecting value`
```python
with open("Test1-CFG.json", 'r') as f:
          data = json.load(f)
```

I try to fix it. When I remove each comma in the list of nodes and edges, I can read the json file correctly.

From
```json
  "nodes": [
    {
      "id": "n1",
      "line": "4",
      "code": "void main(String[] args)"
    },
    {
      "id": "n10",
      "line": "14",
      "code": "System.out.println(\"End\")"
    },
  ],
 "edges": [
    {
      "id": "e1",
      "source": "n1",
      "target": "n2",
      "label": ""
    },
    {
      "id": "e2",
      "source": "n2",
      "target": "n3",
```

to 
```json
  "nodes": [
    {
      "id": "n1",
      "line": "4",
      "code": "void main(String[] args)"
    },
    {
      "id": "n10",
      "line": "14",
      "code": "System.out.println(\"End\")"
    }
  ],
 "edges": [
    {
      "id": "e1",
      "source": "n1",
      "target": "n2",
      "label": ""
    },
    {
      "id": "e2",
      "source": "n2",
      "target": "n3",
```

Can you fix this in the code? 
I think it is maybe in [nodes](https://github.com/ghaffarian/progex/blob/master/src/main/java/ghaffarian/progex/graphs/ast/AbstractSyntaxTree.java#L176) and [edges](https://github.com/ghaffarian/progex/blob/master/src/main/java/ghaffarian/progex/graphs/ast/AbstractSyntaxTree.java#L192)

If it is a last node or a last edge, it does not have to write ,(comma)

**Comment by ghaffarian** [comment]
I will check this out and return with an answer ASAP.

**Comment by ghaffarian** [comment]
I pushed a fix for this issue, and this is now fixed in master.

It will take me some more time (maybe 10 days) to publish a new release with the latest changes.

**Comment by ghaffarian** [comment]
I just released PROGEX-v3.4.0 which includes the fix for this issue and some other improvements.
